test/acvp_test.c

```diff
@@ -357,6 +360,9 @@ static int ecdh_cofactor_derive_test(int tstid)
     int use_cofactordh = t->key_cofactor;
     int cofactor_mode = t->derive_cofactor_mode;
 
+    if (!ec_cofactors)
```

Comment:
If OPENSSL_NO_EC is defined, ec_cofactors won't be declared.  Won't the build break here in that case?

Reply:
Line 139 has the guarding ifdef.